package.json

```diff
@@ -79,7 +79,7 @@
     "test:regressions-pigment-css:run": "nx run nx_test_regressions_pigment_css_run",
     "test:regressions-pigment-css:server": "pnpm --filter @app/pigment-css-vite-app run preview --port 5001",
     "test:unit": "nx run nx_test_unit",
-    "test:argos": "node ./scripts/pushArgos.mjs",
+    "test:argos": "node ./scripts/pushArgos.mjs test/regressions/screenshots/chrome",
```

Comment:
WDYT about making this the default argument?
I imagine that most if not all repos/projects will use the same path. 🤔

Reply:
I don't really mind adding a default, but `test/regressions/screenshots/chrome`, while currently being in use, feels a bit nonsensical to me.

Reply:
I suggested it only because the migration would be seamless. 🙈 
But yeah, I agree, we could omit the `chrome` part, since we do not create screenshots on other browsers anyway... 🤷

Reply:
Sure, I have no objections to this approach as well. 👍 
It will be cleaner long term. 👌 
We just need to remember that this is a breaking change that will need adjustment when `@mui/monorepo` is bumped. ℹ️